tests/Unit/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/Test_GaussianPlusConstant.cpp

```diff
@@ -0,0 +1,115 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cmath>
+#include <cstddef>
+#include <limits>
+#include <random>
+
+#include "DataStructures/DataVector.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GaussianPlusConstant.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/GhDampingFunction.hpp"
+#include "Framework/SetupLocalPythonEnvironment.hpp"
+#include "Framework/TestCreation.hpp"
+#include "Framework/TestHelpers.hpp"
+#include "Helpers/Evolution/Systems/GeneralizedHarmonic/GhDampingFunctions/TestHelpers.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "Utilities/Gsl.hpp"
+
+template <size_t VolumeDim, typename Fr>
+class GhDampingFunction;
+
+namespace Frame {
+struct Grid;
+struct Inertial;
+}  // namespace Frame
+
+namespace {
+template <size_t VolumeDim, typename DataType, typename Fr>
+void test_gaussian_plus_constant_random(
+    const DataType& used_for_size) noexcept {
+  Parallel::register_derived_classes_with_charm<
+      GhDampingFunctions::GaussianPlusConstant<VolumeDim, Fr>>();
+
+  // Generate the amplitude and width
+  MAKE_GENERATOR(gen);
+  std::uniform_real_distribution<> real_dis(-1, 1);
+  std::uniform_real_distribution<> positive_dis(0, 1);
+
+  const double constant = real_dis(gen);
+  const double amplitude = positive_dis(gen);
+  // If the width is too small then the terms in the second derivative
+  // can become very large and fail the test due to rounding errors.
+  const double width = positive_dis(gen) + 0.5;
+
+  // Generate the center
+  std::array<double, VolumeDim> center{};
+  for (size_t i = 0; i < VolumeDim; ++i) {
+    gsl::at(center, i) = real_dis(gen);
+  }
+
+  GhDampingFunctions::GaussianPlusConstant<VolumeDim, Fr> gauss_plus_const{
+      constant, amplitude, width, center};
+
+  TestHelpers::GhDampingFunctions::check(
+      std::move(gauss_plus_const), "gaussian_plus_constant", used_for_size,
+      {{{-1.0, 1.0}}}, constant, amplitude, width, center);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE(
```

Comment:
can you combine the two test cases into one by turning the test cases into functions and having a single test case call the two functions? We're trying to reduce the number of individual test cases